### ISSUE TITLE
Crash when joining server

### ISSUE THREAD

**Issue by darkhax** [McJty/InControl#31]
MC: 1.12.1
Version: 3.6.0

```
java.lang.NoSuchMethodError: net.minecraft.world.biome.Biome.func_185359_l()Ljava/lang/String;
	at mcjty.incontrol.rules.support.GenericRuleEvaluator.lambda$addBiomesCheck$27(GenericRuleEvaluator.java:396)
	at mcjty.incontrol.rules.support.GenericRuleEvaluator.match(GenericRuleEvaluator.java:636)
	at mcjty.incontrol.rules.PotentialSpawnRule.match(PotentialSpawnRule.java:170)
	at mcjty.incontrol.ForgeEventHandlers.onPotentialSpawns(ForgeEventHandlers.java:94)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1110_ForgeEventHandlers_onPotentialSpawns_PotentialSpawns.invoke(.dynamic)
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:179)
	at net.minecraftforge.event.ForgeEventFactory.getPotentialSpawns(ForgeEventFactory.java:245)
	at net.minecraft.world.WorldServer.func_175734_a(WorldServer.java:244)
	at net.minecraft.world.WorldEntitySpawner.func_77192_a(WorldEntitySpawner.java:127)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:202)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:754)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:396)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:666)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:524)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by McJty** [comment]
I'll check it out soon

**Comment by McJty** [comment]
Fixed in 3.6.1 which is waiting for approval on curseforge

**Comment by darkhax** [comment]
Thanks :)
